src/source/feeds.rst

```diff
@@ -31,6 +82,701 @@ XML file.
    Any change in the ad (including it re-appearing in the feed if it wasn't present the previous time) will update the
    ad and trigger image re-processing.
 
+======= ====================================================
+Example	.. code-block:: html
+        
+          <ad:vendorId>15839942</ad:vendorId>
+======= ====================================================
+
+.. index:: externalId
+.. _feed_externalId:
+
+externalId
+""""""""""
+.. warning::
+   There is still an **externalId** field in the XSD, this field is replaced by **vendorId**.
+   Please update your XML to reflect this change. This makes naming consistent between feeds and sellside API.
+   The **vendorId** field in the feeds has the same meaning and constraints as the **vendorId** field in the
+   sellside API.
+
+.. index:: sellerName
+.. _feed_sellerName:
+
+sellerName
+""""""""""
+
+Display name of the seller (max. 60 characters).
+
+======= ======================================================
+Example	.. code-block:: html
+        
+         <ad:sellerName>Cups, Caps &amp; Craps</ad:sellerName>
+======= ======================================================
+
+.. index:: title
+.. _feed_t:
+
+title
+"""""
+
+Any string, with minimum and maximum length determined by the category, with a maximum cap of 1024 characters. See :ref:`categories`. URLs are not allowed as part of the title.
+
+======= ====================================================
+Example	.. code-block:: html
+        
+         <ad:title>Goedkope A-merk herenfietsen</ad:title>
+======= ====================================================
+
+.. index:: description
+.. _feed_descr:
+
+description
+"""""""""""
+
+The description field of the ad.
+Any string, with minimum and maximum length determined by the category, with a maximum cap of 1MB characters. See :ref:`categories`. URLs are not allowed as part of the description.
```

Comment:
nit: it's weird to express the title max cap as an integer, and here as another unit (MB)

Reply:
also, are you sure of this? All I see is 

```
| description           | text                | NO   |     | NULL    |                |
```

Reply:
looks like way less if I look here https://dev.mysql.com/doc/refman/5.6/en/storage-requirements.html#data-types-storage-reqs-strings

Reply:
description length and title length are defined by the tenant in the taxonomy, which will be an additional validation step on top of the XML structure. The XML structure can allow for 1MB of description, but at a later step we will probably cut that to 64K.

Reply:
ok, then writing about this 1 MB is indeed odd. I will remove it.